### PR TITLE
chore: upgrade to Aspect CLI 5.8.5

### PR DESCRIPTION
**Pull request by gregmagolan**
The 5.8.5 release updates `configure` to build with the latest gazelle (v0.33.0) under the hood

## Test plan

Ran `aspect configure` locally
CI

### CODE REVIEW

File: client/eslint-plugin-wildcard/BUILD.bazel [hunk 1]
```diff
@@ -3,6 +3,7 @@ load("//dev:defs.bzl", "npm_package")
 load("@aspect_rules_js//js:defs.bzl", "js_library")
 
 # gazelle:js_files **/*.js
+# gazelle:js_ignore_imports ./rules/forbid-class-name
```
Comment (jbedard) [inline client/eslint-plugin-wildcard/BUILD.bazel:6]: I think just delete the `js_library` and let gazelle re-generate it. Previously it was done manually but can now be automated.

File: client/observability-server/BUILD.bazel [hunk 2]
```diff
@@ -42,6 +42,7 @@ ts_project(
         "//:node_modules/lodash",
         "//:node_modules/node-fetch",
         "//:node_modules/signale",
+        "//client/observability-client:observability-client_lib",
```
Comment (jbedard) [inline client/observability-server/BUILD.bazel:45]: This is one I'm uncertain about and wonder if we need a way to opt-out or do this better. Sometimes it's "correct", other times (like maybe this one) it is not ideal.

Essentially there is a `declare mode "foo-lib"` in `client/observability-client-client_lib` (the `ts_project` compiling all the sources), and this package (`observability-server`) does `import "foo-lib"`. If `observability-server` is the correct definition of this module then that is correct, if `observability-server` is only extending it in a way that should only be local then maybe we wouldn't want this line...

File: client/browser/BUILD.bazel [hunk 3]
```diff
@@ -277,7 +277,6 @@ ts_project(
         "//:node_modules/utility-types",
         "//:node_modules/uuid",
         "//:node_modules/webext-domain-permission-toggle",
-        "//:node_modules/webextension-polyfill",
```
Comment (gregmagolan) [inline client/browser/BUILD.bazel:280]: Looks like this one should have been kept as it causes a build failure to remove. Any idea what changed @jbedard ?

Comment (jbedard) [reply client/browser/BUILD.bazel:280]: I assume related to the `declare module 'webextension-polyfill'` within `client/browser`. But that declaration should be treated as a type and be an additional dep, not a replacement to this one.

Maybe log an issue and add a `#keep` for now?

**Review by burmudar — APPROVED** [review]

**Pull request merged**